`subliminal-test` can now be run alongside the Xcode 5 toolchain, and respects the "workspace" argument.

wearhere opened the pull request.

Commit:
`set_simulator_device` is now a shell script.

For increased reliability.

Commit:
`subliminal-test` now properly builds a workspace if one is specified. (fixes #31)

wearhere commented:
This fixes both #31 and #33 because I couldn't test the former without the latter.

Commit:
Provide for `subliminal-test` to use the Xcode 4.6 toolchain
while 5 support is pending. (fixes #33, refs #32)

This entails setting the Xcode directory to that of Xcode 4.6 before launching
`subliminal-test`, as described in the documentation, as well as forcing the CI scripts
to open the Simulator corresponding to the currently selected dev toolchain: the
Xcode directory in-and-of-itself has no effect on which "iPhone Simulator" is launched.

wearhere commented:
For anyone keeping score, the attempt to convert `reset_simulator` into a text document (9f54bc83dc6d110dbe840cda5a35cbab60542d2b) [failed](https://travis-ci.org/inkling/Subliminal/builds/8333273#L1104) and so was removed. :cry:

wearhere commented:
Bloody hell, `reset_simulator` timed out again even as a compiled script. I'll look into replacing it with a programmatic solution tomorrow.

HaloZero commented:
@wearhere where was the script timing out? I felt like adding a delay between application launch and trying to set the menu option would work.

MaxGabriel commented on README.md:
````diff
@@ -277,9 +277,22 @@ application and tests.
+
+```sh
+export DEVELOPER_DIR=/Applications/Xcode.app/Contents/Developer
+```
````
Is this different from `xcode-select`?

wearhere replied:
It's documented in `xcode-select`'s `man` page as having the same effect as `xcode-select --switch <path>`--but only within the same shell & subshells, so devs can use the Xcode 5 toolchain normally if they like. The other benefit vs. `xcode-select --switch` is that setting `DEVELOPER_DIR` doesn't require superuser privileges.

MaxGabriel commented:
Yep, this works perfectly:

[x] On a workspace with only Xcode 4.6 installed
[x] On a workspace with both Xcode 4.6 and Xcode 5 installed
[x] On a regular project, with both Xcode 4.6 and Xcode 5 installed

Commit:
`reset_simulator` is now a shell script.

For increased reliability.

wearhere commented:
@HaloZero It's difficult to say where the script was timing out because Applescript errors [don't include line-numbers](https://travis-ci.org/inkling/Subliminal/builds/8333273#L1104) (at least not in a form which corresponds to the decompiled source). Your guess is reasonable. I don't think that it's worth debugging when we can reset the content and settings directly (fbb4ddf), though--I'd rather not worry about timing issues across different environments (to say nothing of the fragility of trying to script applications' interfaces anyway).

@MaxGabriel Thanks for running those tests. @inkling/committers, this is ready for review.

wearhere commented:
Lol that's not the right group. And it appears that I don't have permissions to create a more appropriate one... ok how about @ahaneyinkling or @aegolden, one of you please.

Commit:
Allow a second more for the Simulator to start up as required by the CI scripts.

To ensure that it is fully open before the next action occurs, particularly instruments launching.
